crates/youki/src/main.rs

```diff
@@ -38,6 +38,11 @@ use nix::unistd::getuid;
 #[derive(Parser, Debug)]
 #[clap(version = crate_version!(), author = "youki team")]
 struct Opts {
+    // I don't know how to get the log level when the --debug flag is not set (I want to show some default values on the help page when the options are not set)
+    // Example: '--debug     change log level to debug. (default: "warn")'
+    /// change log level to debug.
```

Comment:
These comments in this section are `./youki -h` will be output.
When I thought about it, I felt that the first line was unnecessary.

Reply:
These codes may be helpful for you.
https://github.com/containers/youki/blob/ba08bb323165f4d0b98a0321fb990725bb32de69/crates/youki/src/commands/create.rs#L10-L27

Reply:
hmm

Reply:
> These codes may be helpful for you.
> 
> https://github.com/containers/youki/blob/ba08bb323165f4d0b98a0321fb990725bb32de69/crates/youki/src/commands/create.rs#L10-L27

The default value in some options is dynamic, for example, `--root`

Reply:
@unknowndevQwQ 
My point is that the comments in this section are not for developers, but for users. I would like you to take that into account when making your comments.

Reply:
> @unknowndevQwQ My point is that the comments in this section are not for developers, but for users. I would like you to take that into account when making your comments.

Should I delete L41-L42?

Reply:
@utam0k